RELEASE/scripts/autoscend/quests/level_13.ash

```diff
@@ -83,6 +83,9 @@ boolean EightBitRealmHandler()
 		case "red":
 			// limited buff that is helpful for 3 of 4 8-bit zones
 			buffMaintain($effect[shadow waters]);
+			if(meat_drop_modifier() < 395){
+				auto_getCitizenZone("meat");
```

Comment:
I think that if we keep meat on and ignore the item buff, this might be the best solution because of the meat buff being useful at the wall of meat too